src/renderers/shared/fiber/ReactDebugCurrentFiber.js

```diff
@@ -33,9 +33,13 @@ function getCurrentFiberOwnerName() : string | null {
     }
     switch (fiber.tag) {
       case IndeterminateComponent:
-      case FunctionalComponent:
       case ClassComponent:
         return getComponentName(fiber);
+      case FunctionalComponent:
```

Comment:
oh, idk. It was just a thing I tried and wanted to document. I tried it in an attempt to not give `FunctionalComopnent`s their own case.